quickstarts/jbpm-workitems-camel/src/main/java/org/jboss/integration/fuse/quickstarts/jbpm/workitems/camel/ApplicationServlet.java

```diff
@@ -0,0 +1,112 @@
+package org.jboss.integration.fuse.quickstarts.jbpm.workitems.camel;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.jbpm.process.workitem.camel.CamelHandler;
+import org.jbpm.process.workitem.camel.CamelHandlerFactory;
+import org.kie.api.KieServices;
+import org.kie.api.runtime.KieContainer;
+import org.kie.api.runtime.KieSession;
+
+public class ApplicationServlet extends HttpServlet {
+
+    int APPLICATION_COUNT = 10; // The number of applications to create at the activation of bundle
+
+    /* Range for mortgage ammount */
+    int MORTGAGE_MIN = 100000;
+    int MORTGAGE_MAX = 1000000;
+
+    /* The range for applicant income */
+    int INCOME_MIN = 20000;
+    int INCOME_MAX = 200000;
```

Comment:
These look like constants, perhaps we could make them `(public|private) static final`?